css/style.css

```diff
@@ -0,0 +1,141 @@
+:root {
+    --color-light: oldlace;
+    --color-dark: #333;
+    --color-primary: rgb(148, 226, 148);
+    --color-primary-dark: rgb(53, 73, 53);
+}
+ 
+ * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Setting the value of 1rem */
+html {
+    /* 1rem = 10 pixels */
+    font-size: 62.5%;
+}
+ 
+body {
+    font-family: "Nanum Gothic", sans-serif;
+    font-size: 1.6rem;
+    background-color: var(--color-light);
+    color: var(--color-dark);
+}
+
+a {
+    text-decoration: none;
+    color: var(--color-dark);
+}
+
+ul {
+    list-style: none;
+}
+
+img {
+    display: block;
+    width: 100%;
+    max-width: 100%;
+}
+
+
+.logo {
+    font-family: 'Inspiration', cursive;
+    font-size: 5rem;
+    color: var(--color-dark);
+    font-weight: 800;
+    margin-left: 7rem;
+}
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    background: var(--color-primary);
+    color: var(--color-light);
+
+    position: fixed;
```

Comment:
**Stylelint Error:** _Expected no more than 1 empty line_
Please remove any empty line of code inside {}.

Reply:
- [ ] I think we can improve :chart_with_upwards_trend:   :chart_with_upwards_trend:  this review message by being more descriptive and giving a reason for the requested change.
> First state the aimed problem ❗, politely offer a suggestion 😏,  conclude with a reason or importance of making that change and if possible use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

Reply:
- [ ] I think we can improve :chart_with_upwards_trend:   :chart_with_upwards_trend:  this review message by being more descriptive and giving a reason for the requested change.
>  State the aimed problem ❗, politely offer a suggestion 😏,  conclude with a reason or importance of making that change, and if possible use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.